src/compiler/checker.ts

```diff
@@ -28011,7 +28011,7 @@ namespace ts {
                 return undefined;
             }
 
-            if (isDeclarationNameOrImportPropertyName(node)) {
+            if (isModifier(node) || isDeclarationNameOrImportPropertyName(node)) {

```

Comment:
Would this affect quick info we show? Can you please verify that?

Reply:
This does affect `quickInfoInvalidLocations.ts` -- previously we would show no quick info at the modifier, now we will.

Reply:
Which is probably not what user expects..  Why cant findAllReferences see if modifier then get symbol of parent?

Reply:
The alternative is not to get any quick info at all, so I think this is a good thing. The user may not be so careful exactly what characters they are hovering over with their mouse or care about the modifier/name distinction, so this could help people see quick info where otherwise they would have tried to get it and failed.